src/components/MainMenu.js

```diff
@@ -1,33 +1,41 @@
-import React from 'react';
+import React from "react";
 import {
   // Button,
   Container,
   Image,
-  Menu,
-} from 'semantic-ui-react';
-import { Link } from 'react-router-dom';
-import { BetaWarning } from './BetaWarning';
-import logo from '../images/logo.svg';
+  Menu
+} from "semantic-ui-react";
+import { Link } from "react-router-dom";
+import { BetaWarning } from "./BetaWarning";
+import logo from "../images/logo.svg";
 
-
-const MainMenu = function ( props ) {
-  return(
+const MainMenu = function(props) {
+  return (
     <Container>
-      <Menu inverted secondary size='large'>
+      <Menu inverted secondary size="large">
+        <Menu.Item>
+          <a
+            href="http://www.codeforboston.org"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <Image src={logo} size="tiny" />
+          </a>
+        </Menu.Item>
         <Menu.Item>
-          <a href="http://www.codeforboston.org" target="_blank" rel="noopener noreferrer"><Image src={logo} size='tiny' /></a>
+          <Link to="/">Home</Link>
```

Comment:
Side-note: These questions may sound stupid, but I'm going for them anyway. Did the linter do this? Is this somehow still linking to the correct page?

Reply:
lol, forget it. The lines of the diff are just off.